config

```diff
@@ -2,17 +2,21 @@
 
 TARGET=${TARGET:-"centos"}
 
-export NAMESPACE=${DOCKER_NAMESPACE:-"rhchestage"}
+export NAMESPACE="rhchestage"
 
-if [ $TARGET == "rhel" ]; then
+if [ "$TARGET" == "rhel" ] || [ "$TARGET" == "rh-integration-test" ]; then
   export DOCKERFILE="Dockerfile.rhel"
   export REGISTRY=${DOCKER_REGISTRY:-"push.registry.devshift.net/osio-prod"}
 else
   export DOCKERFILE="Dockerfile"
   export REGISTRY=${DOCKER_REGISTRY:-"push.registry.devshift.net"}
 fi
 
-export DOCKER_IMAGE="rh-che-server"
+if [ "$TARGET" == "rh-integration-test" ]; then
+  export DOCKER_IMAGE="rh-che-automation"
```

Comment:
This is not critical but ideally I would love to have something like `rh-che-server:<upstream-tag>PR#<pr-number>` as image name and tag:
- `DOCKER_IMAGE="rh-che-server"` (the result of the build is always a che server)
- `RH_TAG_DIST_SUFFIX="PR#1234"` (when it's a PRCheckBuild)

Reply:
Will add this as an issue for further improvement, but don't you think it's better to keep the PR test builds separate from normal releases?
For example the registry could have a number of builds to hold set, so it auto-cleans. There will be ~~many~~ builds in the future.

Reply:
Ok to do it in a separate PR. 

Anyway I still believe we should use `DOCKER_IMAGE="rh-che-server"`. 

I would understand if this images would be published to the docker hub where we access through the GUI and can list all the tags of a given image. But do anyone of us really look at the tags of a devshift image? Yes you are right: tomorrow instead of having 100 tags we will probably have 10K tags. But is it really a problem?